src/makers/clodpi/clodpi-pro.svg

```diff
@@ -0,0 +1,716 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This image is pretty hard to see, it might need thinker strokes?
![Screen Shot 2021-07-23 at 6 11 28 PM](https://user-images.githubusercontent.com/6345962/126853385-0ddc1ee6-5ee3-480d-88b0-7063f140f260.png)

Reply:
@tracyharton Updated image.

Reply:
Looks great now